frontend/demo/src/reducers/data/messages/index.ts

```diff
@@ -51,6 +50,18 @@ export default function messagesReducer(state = initialState, action: Action): a
         };
       }
 
+    case getType(actions.sendMessagesAction):
+      return {
+        ...state,
+        all: {
+          ...state.all,
+          [action.payload.conversationId]: {
+            ...state.all[action.payload.conversationId],
+            ...organiseMessages([action.payload.message]),
```

Comment:
Not sure, but this kind looks like all the old messages in the conversation are overridden with the one from the message payload?